src/VisualStudio/Setup/Roslyn.VisualStudio.Setup.csproj

```diff
@@ -228,13 +228,6 @@
       <IncludeOutputGroupsInVSIXLocalOnly></IncludeOutputGroupsInVSIXLocalOnly>
       <Private>false</Private>
     </ProjectReference>
-    <ProjectReference Include="..\Setup.ServiceHub\x64\Roslyn.VisualStudio.ServiceHub.Setup.x64.csproj">
```

Comment:
>   <ProjectReference Include="..\Setup.ServiceHub\x64\Roslyn.VisualStudio.ServiceHub.Setup.x64.csproj"> [](http://example.com/codeflow?start=1&length=103)

It doesn't look like this change belongs to this PR

Reply:
indeed.  that's outright terrifying that VS made that change.